### PR TITLE
Display list file received share for tablet

### PR DESCRIPTION
**Pull request by datnguyenhuy**
https://user-images.githubusercontent.com/56970970/115862146-f4265a00-a45d-11eb-9ffc-8b8acdef3a03.mp4

### CODE REVIEW

**Comment by datnguyenhuy** [comment]
https://user-images.githubusercontent.com/56970970/115860918-61d18680-a45c-11eb-9b0d-5c7c8cd2e115.mp4

File: lib/presentation/widget/received/received_share_widget.dart [hunk 1]
```diff
@@ -242,11 +243,38 @@ class _ReceivedShareWidgetState extends State<ReceivedShareWidget> {
...
+                    flex: 2,
+                  ),
+                  Expanded(
+                    child: _buildSenderName(receivedShareItem.element.sender.fullName()),
+                    flex: 1,
+                  ),
+                  Expanded(
+                    child: _buildModifiedDateText(AppLocalizations.of(context).item_created_date(
+                        receivedShareItem.element.creationDate.getMMMddyyyyFormatString())),
+                    flex: 1,
```
Comment (imGok) [inline lib/presentation/widget/received/received_share_widget.dart:270]: `flex: 1` is not needed since it's the default value

**Comment by imGok** [comment]
image
Suggestions : 

- Can you add a space for too long name before the persons name (with triple dot maybe) ?
- Last name should be in upper case like the mockup 
image

File: lib/presentation/widget/received/received_share_widget.dart (hunk 1 above)
Comment (datnguyenhuy) [reply lib/presentation/widget/received/received_share_widget.dart:270]: I fixed it. Thank you

File: lib/presentation/widget/received/received_share_widget.dart [hunk 2]
```diff
@@ -242,11 +243,38 @@ class _ReceivedShareWidgetState extends State<ReceivedShareWidget> {
...
+                  )
+                ],
+              )
+          ),
         ),
-        subtitle: Transform(
+        subtitle: ResponsiveWidget.isSmallScreen(context)
+            ? Transform(
           transform: Matrix4.translationValues(-16, 0.0, 0.0),
```
Comment (huynguyennovem) [inline lib/presentation/widget/received/received_share_widget.dart:278]: pls fix indention

Comment (datnguyenhuy) [reply lib/presentation/widget/received/received_share_widget.dart]: I fixed it and push commit fixup. Thank you, please

**Review by imGok — APPROVED** [review]

**Pull request closed without merging**